HelpSource/Classes/Function.schelp

```diff
@@ -30,6 +30,15 @@ code::
 ((lambda (x) x) 1) // Lisp
 ::
 
+NOTE:: 
+In case you already know how to define your own classes, you came accross the usage
+of teletype::^:: within methods to return values back to the caller. Please be aware
+that usage of teletype::^:: in functions not part of classes is not implemented.
+and will most likely behave in a non-intuitive way!
+
+Consider using the teletype::block:: method in case you really need the semantic of teletype::^:: in this context.
```

Comment:
I don't think we need the word "really" here. Here's a suggestion (feel free to adjust it):

"Consider using link::Classes/Function#-block:: if you need to return early from the function."

(please check if the link works correctly, I haven't tested it)

Also, I'm not sure if that sentence would work better inside or outside of the `NOTE` block.